### ISSUE TITLE
[Release]: 0.2.3

### ISSUE THREAD

**Issue by BFalquet** [insightsengineering/chevron#669]
### PRs

- [x] https://github.com/insightsengineering/chevron/pull/646

### Issues

- [x] https://github.com/insightsengineering/chevron/issues/668

### Pre-requisites

- [x] Make sure that high priority bugs (label "priority" + "bug") have been resolved before going into the release.
- [x] Review old/hanging PRs before going into the release.
- [x] Revisit R-package's lifecycle badges (Optional).
- [x] Discuss package dependencies before going into release activities.
- [x] Create a plan to sequentially close release activities and submit groups of packages for internal validation (Applicable only for regulatory release).
- [x] Make sure all relevant integration tests are green 2-3 days before the release. Look carefully through logs (check for warnings and notes).
- [x] Check if a package is installable on our supported internal systems (Optional).
- [x] Inform about the soft code freeze, decide what gets merged in before starting release activities.

### Release Checklist

- [x] Recurring tasks: Monitor integration tests, if integration fails, create priority issues on the board.
- [x] Update NEWS.md file: make sure it reflects a holistic summary of what has changed in the package, check README.
- [x] Remove the additional fields (`Remotes` and `Config/Needs/*`) from the DESCRIPTION file where applicable.
- [x] Make sure that the minimum dependency versions are updated in the DESCRIPTION file for the package.
- [x] Increase versioned dependency on 
  - [x] {dunlin} to >=0.1.5.
  - [x] {formatters} >= 0.5.3
  - [x] {rlistings} >= 0.2.5
  - [x] {rtables} >= 0.6.4
  - [x] {tern} >= 0.9.1
- [x] Create a pull request to make necessary bug fixes/changes (add "[skip vbump]" in the PR title), ~and after merging the PR~, tag the update(s) as a release candidate v0.2.3-rc < release candidate iteration > on the main branch. Note that tags are created in GitHub and synchronized with GitLab automatically.
- [x] The package is submitted for internal validation by Release Coordinator (Applicable only for regulatory release).
- [x] Address any feedback (internal validation/user testing), retag the package as a release candidate v0.2.3-rc(n+1). Repeat the submission for internal validation if necessary.
- [x] Get the package validated (Applicable only for regulatory release).
- [x] If the additional fields were removed, add them back in a separate PR, and then merge the PR back to main (add "[skip vbump]" in the PR title).
- [x] Create a git tag with the final version set to 0.2.3 on the main branch.
- [x] Update downstream package dependencies to (>=0.2.3) in {citril}.

### Testing

- [x] Integration tests results - accepted.
- [x] UAT results - accepted.
- [x] ~Shiny apps test results - accepted (Applicable only for Shiny apps).~ Not applicable
- [ ] Necessary testing on target environment - performed (up to ETL).

### Release Feedback

- [ ] Fix 1
- [ ] Enhancement 1
- [ ] Defect 1

### Post-release Checklist

- [x] Make sure that the package is published to internal repositories (Validated and/or Non-Validated repository).
- [x] Review and update installation instructions for the package if needed.
- [x] Verify if a new dev version (.9XXX) has been added to the NEWS.md file and DESCRIPTION file as a placeholder for release notes by automation.
- [x] Make sure internal documentation/documentation catalogs are up to date.
- [x] Notify the IDR team to start post-release/clean-up activities.
- [x] Announce the release on ________.

### Decision tree

Click [here](https://github.com/insightsengineering/.github/blob/main/.github/ISSUE_TEMPLATE/RELEASE_DECISION_TREE.md) to see the release decision tree.
